dockerfiles/Dockerfile.e2e

```diff
@@ -39,6 +39,7 @@ ARG GITCOMMIT
 ENV VERSION=${VERSION}
 ENV GITCOMMIT=${GITCOMMIT}
 ENV DOCKER_BUILDKIT=1
+#ENV GO111MODULE=off
```

Comment:
question: Should we keep this commented line?

Reply:
Arf, nope, my mistake. Let me fix